promql/ast.go

```diff
@@ -197,14 +213,18 @@ func (*StringLiteral) expr()  {}
 func (*UnaryExpr) expr()      {}
 func (*VectorSelector) expr() {}
 
-// VectorMatchCardinaly describes the cardinality relationship
+// VectorMatchCardinality describes the cardinality relationship
 // of two vectors in a binary operation.
 type VectorMatchCardinality int
 
 const (
+	// CardOneToOne represents one-to-one vector matching cardinality.
 	CardOneToOne VectorMatchCardinality = iota
+	// CardManyToOne represents many-to-one vector matching cardinality.
 	CardManyToOne
+	// CardOneToMany represents one-to-many vector matching cardinality.
 	CardOneToMany
+	// CardManyToMany represents many-to-many vector matching cardinality.
 	CardManyToMany
```

Comment:
Essentially all changes in this file are code-obfuscations to me. Them coming from a tool doesn't make them universally correct.
Especially this one – each comment is the type and constant name again and again.